nixos/modules/programs/wayland/labwc.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.labwc;
+in
+{
+  options.programs.labwc = {
+    enable = lib.mkEnableOption (lib.mdDoc "labwc");
+    package = lib.mkPackageOption pkgs "labwc" { };
+  };
+
+  config = lib.mkIf cfg.enable (lib.mkMerge [
+    {
+      environment.systemPackages = [ cfg.package ];
+
+      xdg.portal.config.labwc.default = lib.mkDefault [ "wlr" "gtk" ];
```

It appears labwc sets XDG_CURRENT_DESKTOP to `wlroots`. See https://github.com/labwc/labwc/blob/785a34e8ad6e693591cbf657b4b8ff50192d0eec/src/config/session.c#L106.
```suggestion
      xdg.portal.config.wlroots.default = lib.mkDefault [ "wlr" "gtk" ];
```